app.rb

```diff
@@ -94,6 +94,32 @@ def organizations_added
   def organizations_removed
     before.organizations - after.organizations
   end
+
+  def broken_terms
+    broken_terms = after.events.select do |event|
+      event[:classification] == "legislative period" && (event[:id] =~ /^(term)\/?\d+/).nil?
```

Comment:
I'm not sure this Regex is doing what you think it's doing. At least it's not clear why you're making the "/" optional (but not the "term").

Either way I think this is a fatally flawed approach, as we don't guarantee that term IDs will be in this format.